recipe/meta.yaml

```diff
@@ -10,11 +10,11 @@ source:
 
 build:
   number: 0
-  skip: true  # [osx or win]
+  skip: true  # [osx]
 
 requirements:
   run:
-    - cuda-nsight 12.0.78               # [not aarch64]
+    - cuda-nsight 12.0.78               # [linux64 or ppc64le]
     - nsight-compute 2022.4.0.15
     - cuda-nvml-dev 12.0.76
     - cuda-nvvp 12.0.90                 # [not aarch64]
```

Comment:
Also noticing this now doesn't match. Should we keep them the same?

Reply:
`cuda-nvvp` is supported on windows, so there is a difference here
https://github.com/conda-forge/cuda-nvvp-feedstock/blob/main/recipe/meta.yaml#L16-L18